src/ext/time/components/Clock.jsx

```diff
@@ -17,14 +22,33 @@ var hoursScale   = d3.scale.linear().domain([0, 11 + 59/60]).range([-90, 270]);
 
 
 var Clock = React.createClass({
+    mixins: [
```

Comment:
The ApiConsumerMixin is Only useful when using a client part making calls to external API